groups/ceu-frontend/resourcegroup.tf

```diff
@@ -0,0 +1,20 @@
+resource "aws_resourcegroups_group" "rg_ceu" {
+  name        = "rg-${var.application}-for-ec2-instances-in-dev-env"
```

Comment:
So we wouldnt want the environment name in the Group name as this would be deployed to multiple environments.
The name should be generic enough that it can be used everywhere e.g. `rg-${var.application}-ec2`